store/store.go

```diff
@@ -1519,7 +1533,13 @@ var download = func(ctx context.Context, name, sha3_384, downloadURL string, use
 		dlSize = float64(resp.ContentLength)
 		pbar.Start(name, dlSize)
 		mw := io.MultiWriter(w, h, pbar)
-		_, finalErr = io.Copy(mw, resp.Body)
+		var limiter io.Reader
+		limiter = resp.Body
+		if limit := GetRateLimit(ctx); limit > 0 {
+			bucket := ratelimit.NewBucketWithRate(limit, 2*int64(limit))
```

Comment:
Curious, why is it 2x the number, and why haven't you use int64 for rate limit type in the config instead of float64?

Reply:
> Curious, why is it 2x the number, and why haven't you use int64 for rate limit type in the config instead of float64?

The config needs to be a string with a unit, but indeed using int64 for the actual rate seems better if that's what we need throughout the code anyway.